website/docs/reference/resource-configs/snowflake-configs.md

```diff
@@ -4,6 +4,21 @@ id: "snowflake-configs"
 description: "Snowflake Configurations - Read this in-depth guide to learn about configurations in dbt."
 ---
 
+| Configuration    | Description    | 
+|------------------|----------------|
+|[Iceberg table format](/reference/resource-configs/snowflake-configs#iceberg-table-format)|The dbt-snowflake adapter supports the Iceberg table format and is available for three of the Snowflake materializations [table](/docs/build/materializations#table), [incremental](/docs/build/materializations#incremental) and [dynamic tables](/reference/resource-configs/snowflake-configs#dynamic-tables)|
+|[Dynamic tables](/reference/resource-configs/snowflake-configs#dynamic-tables)|Specific to Snowflake but follows the implementation of [materialized views](/docs/build/materializations#Materialized-View).|
+|[Temporary tables](/reference/resource-configs/snowflake-configs#temporary-tables)|The `tmp_relation_type` configuration lets you opt into using temporary tables for incremental builds.|
+|[Transient tables](/reference/resource-configs/snowflake-configs#transient-tables)|Transient tables allow time travel for 1 day, with no fail-safe period. By default, dbt creates all Snowflake tables as transient.|
+|[Query tags](/reference/resource-configs/snowflake-configs#query-tags)|Snowflake parameter that can be quite useful when searching in the `QUERY_HISTORY` view|.
+|[Merge behavior (incremental models)](/reference/resource-configs/snowflake-configs#merge-behavior-incremental-models)|The `incremental_strategy` config determines how dbt builds incremental models. By default, dbt uses a merge statement on Snowflake to refresh these tables. The Snowflake adapter supports the following incremental materialization strategies &mdash; `append`, `delete+insert`, `insert_overwrite`, `merge` and [`microbatch`](/docs/build/incremental-microbatch).|
+|[`cluster_by`](/reference/resource-configs/snowflake-configs#configuring-table-clustering)|Use the `cluster_by` config to control clustering for a table or incremental model. It orders the table by the specified fields and adds the clustering keys to the target table.|
```

Comment:
this doesn't include automatic_clustering -- should it? this is tied to my previous feedback about consolidating some of the rows so it matches the headers?